src/biogeochem/CNGapMortalityMod.F90

```diff
@@ -139,7 +149,44 @@ subroutine CNGapMortality (bounds, num_soilc, filter_soilc, num_soilp, filter_so
          laisun     =>    canopystate_inst%laisun_patch  , & ! Input:  [real(r8) (:)   ]  sunlit projected leaf area index      
          laisha     =>    canopystate_inst%laisha_patch  , & ! Input:  [real(r8) (:)   ]  shaded projected leaf area index   
                                                          
-         nind       =>    dgvs_inst%nind_patch         & ! Output: [real(r8) (:) ]  number of individuals (#/m2) added by F. Li and S. Levis
+         nind       =>    dgvs_inst%nind_patch,         & ! Output: [real(r8) (:) ]  number of individuals (#/m2) added by F. Li and S. Levis		
```

Comment:
Please align all the associate lines here in this file and other files you edited. Please add the intent, dimension, etc. of input output as a comment to all these lines consistent with the original code.